### ISSUE TITLE
Problem with the build Python 3.4.3 on Mac

### ISSUE THREAD

**Issue by kornev** [yyuu/pyenv#386]
Python 2.7.8 build fine, but not python 3.4.3...

Version:
pyenv 20150601
Mac OS 10.9.5

I run:
CFLAGS="-I$(xcrun --show-sdk-path)/usr/include" pyenv install -v 3.4.3

```
Successfully installed pip-6.0.8 setuptools-12.0.5
dyld: lazy symbol binding failed: Symbol not found: _fdopendir$INODE64
  Referenced from: /private/var/folders/9_/cw151cf55yv5k3y52_y06tl40000gn/T/python-build.20150605004637.27721/Python-3.4.3/./python.exe
  Expected in: /usr/lib/libSystem.B.dylib

dyld: Symbol not found: _fdopendir$INODE64
  Referenced from: /private/var/folders/9_/cw151cf55yv5k3y52_y06tl40000gn/T/python-build.20150605004637.27721/Python-3.4.3/./python.exe
  Expected in: /usr/lib/libSystem.B.dylib

/bin/sh: line 1: 38300 Trace/BPT trap: 5       ./python.exe -E -m ensurepip $ensurepip --root=/
make: *** [install] Error 133

BUILD FAILED (OS X 10.9.5 using python-build 20150519)

Inspect or clean up the working tree at /var/folders/9_/cw151cf55yv5k3y52_y06tl40000gn/T/python-build.20150605004637.27721
Results logged to /var/folders/9_/cw151cf55yv5k3y52_y06tl40000gn/T/python-build.20150605004637.27721.log

Last 10 log lines:
dyld: lazy symbol binding failed: Symbol not found: _fdopendir$INODE64
  Referenced from: /private/var/folders/9_/cw151cf55yv5k3y52_y06tl40000gn/T/python-build.20150605004637.27721/Python-3.4.3/./python.exe
  Expected in: /usr/lib/libSystem.B.dylib

dyld: Symbol not found: _fdopendir$INODE64
  Referenced from: /private/var/folders/9_/cw151cf55yv5k3y52_y06tl40000gn/T/python-build.20150605004637.27721/Python-3.4.3/./python.exe
  Expected in: /usr/lib/libSystem.B.dylib

/bin/sh: line 1: 38300 Trace/BPT trap: 5       ./python.exe -E -m ensurepip $ensurepip --root=/
make: *** [install] Error 133
```

**Comment by yyuu** [comment]
Umm. Looks weird.

Please create a gist from the complete output from `pyenv install -v 3.4.3`.

**Comment by kornev** [comment]
gist for: pyenv install -v 3.4.3
https://gist.github.com/kornev/b1734dc73667a779bc09

Error with this command associated with empty SSL library in path.
I added CFLAGS and get new error (up).

gist for: CFLAGS="-I$(xcrun --show-sdk-path)/usr/include" pyenv install -v 3.4.3
https://gist.github.com/kornev/e6dbabbaa9ae64ba5e03

**Comment by filipeximenes** [comment]
+1

**Comment by yyuu** [comment]
Your problem looks similar to #263, #273 and #279. You might have some problem in your installation of Command Line Tools for Xcode. Please review your system and try again.

**Comment by yyuu** [comment]
In #429, now I'm trying to add functionality to install `openssl` if it's not found in `/usr/include` or brew's prefix. It should mitigate the issue like this.

**Comment by yyuu** [comment]
I merged changes in #429 into master branch. With latest master, python-build should build & install `openssl` if it is not available via both Xcode and Homebrew. This should solve the issue of missing OpenSSL library intrinsically.

Feel free to reopen if you still have some issues.
